packages/autorest.typescript/src/generators/samples/rlcSampleGenerator.ts

```diff
@@ -1,371 +1,490 @@
-import { ExampleParameter, TestCodeModel } from "@autorest/testmodeler/dist/src/core/model";
+import {
+  ExampleParameter,
+  TestCodeModel
+} from "@autorest/testmodeler/dist/src/core/model";
 import { Project } from "ts-morph";
 import { getAutorestOptions, getSession } from "../../autorestSession";
-import * as fs from 'fs';
-import * as path from 'path';
+import * as fs from "fs";
+import * as path from "path";
 import * as hbs from "handlebars";
 import { NameType, normalizeName } from "../../utils/nameUtils";
 import { getLanguageMetadata } from "../../utils/languageHelpers";
 import { transformBaseUrl } from "../../transforms/urlTransforms";
-import { PathMetadata, Paths, RLCSampleDetail, RLCSampleGroup, SampleParameter, SampleParameters, TestSampleParameters, OperationMethod } from "../../restLevelClient/interfaces";
+import {
+  RLCSampleDetail,
+  RLCSampleGroup,
+  SampleParameter,
+  SampleParameters,
+  TestSampleParameters,
+  OperationMethod
+} from "../../restLevelClient/interfaces";
 import { camelCase } from "@azure-tools/codegen";
 import { pathDictionary } from "../../restLevelClient/generateClientDefinition";
 import { Operation, ParameterLocation } from "@autorest/codemodel";
 import { isLongRunningOperation } from "../../restLevelClient/helpers/hasPollingOperations";
 import { isPagingOperation } from "../../utils/extractPaginationDetails";
 import { getSecurityInfoFromModel } from "../../utils/schemaHelpers";
 import { getParameterAssignment } from "../../utils/valueHelpers";
+import { Paths, PathMetadata } from "@azure-tools/rlc-codegen";
```

Comment:
looks like the sample generation refactor is inevitable ?

Reply:
I think the main change is that 

- `Paths` models changed from `autorest.typescript` to `rlc-codegen`
- Format changes

Right?

Reply:
@qiaozha yeah, I had to do it so we consume the common interfaces across the board
@MaryGao you're right!